org.dpsoftware.FireflyLuciferin.appdata.xml

```diff
@@ -0,0 +1,75 @@
+    <developer_name>Firefly Luciferin</developer_name>
+    <summary>Ambilight clone</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://github.com/sblantipodi/firefly_luciferin?tab=GPL-3.0-1-ov-file</project_license>
+    <url type="homepage">https://github.com/sblantipodi/firefly_luciferin</url>
+    <url type="help">https://github.com/sblantipodi/firefly_luciferin/wiki</url>
+    <description>
+        <p>Luciferin is a generic term for the light-emitting compound found in organisms that generate bioluminescence like Fireflies and Glow Worms.
+            Firefly Luciferin is a Java Fast Screen Capture PC software designed for Glow Worm Luciferin firmware,
+            those two software create the perfect Bias Lighting and Ambient Light system for PC.
+            Differently from other solutions, Firefly and Glow Worm Luciferin are designed to work together for the best experience possible.</p>
+        <p>Key features</p>
+        <ul>
+            <li>Best in class performance combined with ultra low CPU/GPU usage.</li>
+            <li>Advanced algorithms for smooth colors transitions and color correction. Seeing is believing.</li>
+            <li>Wireless or cabled, local or remote? Choose your flavour with MQTT support and Home Assistant integration.</li>
+            <li>Multi monitor support with single or multiple instances.</li>
+            <li>Surround lighting with satellites.</li>
+            <li>Web browser firmware installer and Web Interface.</li>
+            <li>Programmable firmware, change your microcontroller's settings on the fly.</li>
+            <li>Frequent updates, upgrade your PC software and your firmware in one click.</li>
+            <li>Automatic switching between aspect ratios based on your video content.</li>
+            <li>Made from a gamer, for gamers. No added lag, stutter free.</li>
+            <li>Multi platform, Windows and Linux ready. macOS is coming when it's ready.</li>
+            <li>If you don't want to design your own PCB and you have a soldering iron, there is Luciferin's official PCB.</li>
+            <li>Have a question? Get answered on the community.</li>
+        </ul>
+    </description>
+    <screenshots>
+        <screenshot type="default">
+            <caption>Ambilight effect</caption>
+            <image type="source">https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/dx12_pipeline/data/img/ambilight_show.png</image>
+        </screenshot>
+        <screenshot type="default">
+            <caption>Artist</caption>
+            <image type="Luciferin UI">https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/master/data/img/settings_1.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Luciferin UI</caption>
+            <image type="source">https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/master/data/img/settings_2.png</image>
```

Comment:
you never reference a screenshot from the tip of a branch

Reply:
can you elaborate this please? what do you mean? what should I do? where should I put that screenshots?
it's not a branch, it's the master branch. how am I supposed to link it?

Reply:
> it's not a branch, it's the master branch.

Contradiction detected.

Reply:
git has this thing called commits. they are immutable. Chances that the commit disappear is only high if the repo is deleted or rewritten from scratch.

Reply:
can you help me here please?
how can I use an image saved on the GitHub repo then?

Reply:
@sblantipodi Use a screenshot link that references a specific commit, instead of referencing the `master` branch, i.e.:

`https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/08852270fe719b1c0ad2aaeade321118ecaa9b59/data/img/settings_2.png`

Reply:
I often update images in that path because they are screenshots of the app. I have an entire wiki that points to the main branch and I never experienced a broken link.
https://github.com/sblantipodi/firefly_luciferin/wiki

pointing to a specific commit, means that I will need to update the xml file every time I update an image.
@guihkx is this really better than pointing to the main branch? if this are the guidelines ok, but I don't consider this a good thing :)

Reply:
I'm not sure if there are written guidelines about this, but it's the general recommendation.

Sure, having to update the commit when the screenshot changes is less convenient than always referring to `master`, but it makes more sense for historical/archiving purposes.

For example, imagine that someone could be installing an older version of your app. If they were to look at the screenshot you linked from `master`, a completely different UI could be shown...

Reply:
that's good point, fixing it thanks